README.md

```diff
@@ -101,6 +101,14 @@ Note: if you do not include the `--config` option with a path to a configuration
 
 To shutdown `mongod` started manually, use the `admin` database and run `db.shutdownServer()`:
 
+ * Shutdown for MongoDB Community Server 5.x and up:
```

Comment:
this should probably say 6.x and up

we shipped a mongo shell application in 5.x, but the next release 6.0, it was then removed, and mongosh became the recommended shell application